Testscripts/Windows/VALIDATE-DA.ps1

```diff
@@ -34,7 +34,7 @@ function Main {
         return "FAIL"
     }
 
-    Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort ($remoteScriptName + "." + $remoteScriptType) -runAsSudo
+    Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort -runMaxAllowedTime 480 -command "sudo bash $remoteScriptName.$remoteScriptType" -runAsSudo
```

Comment:
Not sure where 480 comes from. A comment about how the value was selected would be helpful for those who will maintain this code in the future.

Reply:
Added comment